### PR TITLE
4296: In Progress for PetitionsClerks

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/presenter/computeds/formattedWorkQueue.js [hunk 1]
```diff
@@ -282,11 +282,17 @@ export const filterWorkItems = ({
...
-            (item.document.isFileAttached === false || item.inProgress)
+            // DocketClerks
+            (item.assigneeId === user.userId &&
+              user.role === USER_ROLES.docketClerk &&
+              !item.completedAt &&
+              item.isQC &&
+              item.section === user.section &&
+              (item.document.isFileAttached === false || item.inProgress)) ||
+            // PetitionsClerks
+            (item.assigneeId === user.userId &&
```
Comment (matthopson) [inline web-client/src/presenter/computeds/formattedWorkQueue.js:293]: This is an interesting change - so all items associated with a case in "in progress" status will show for petitions clerk? Regardless of the item section or completion status, etc..?

Comment (matthopson) [reply web-client/src/presenter/computeds/formattedWorkQueue.js:293]: Also missing tests for this.

File: web-client/src/presenter/computeds/workQueueHelper.js [hunk 2]
```diff
@@ -78,15 +78,18 @@ export const workQueueHelper = (get, applicationContext) => {
...
     showIndividualWorkQueue,
     showMessageContent: !isDisplayingQC,
     showMessagesSentFromColumn: !isDisplayingQC,
     showMyQueueToggle:
       workQueueIsInternal || userIsDocketClerk || userIsPetitionsClerk,
     showOutbox,
-    showProcessedByColumn: isDisplayingQC && userIsDocketClerk && showOutbox,
+    showProcessedByColumn:
+      (isDisplayingQC && userIsDocketClerk && showOutbox) ||
+      (userIsPetitionsClerk && showInProgress),
```
Comment (matthopson) [inline web-client/src/presenter/computeds/workQueueHelper.js:89]: Could we add test coverage for these values?

Comment (ericsorenson) [reply web-client/src/presenter/computeds/workQueueHelper.js:89]: D'oh, good point. Completely forgot.

Comment (matthopson) [reply web-client/src/presenter/computeds/workQueueHelper.js:89]: I think the last person that touched this forgot too :P

File: web-client/src/presenter/computeds/formattedWorkQueue.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/presenter/computeds/formattedWorkQueue.js:293]: I believe so, there's a very limited window for the case to be in this state. I'll check again with Amy.

Comment (matthopson) [reply web-client/src/presenter/computeds/formattedWorkQueue.js:293]: Yeah, just seems really broad for the items that would match this vs the other scenarios

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/computeds/formattedWorkQueue.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/presenter/computeds/formattedWorkQueue.js:293]: We reviewed with Amy, the criteria look accurate.

**Review by matthopson — APPROVED** [review]

**Pull request merged**